otter/test/test_cloud_client.py

```diff
@@ -405,3 +414,162 @@ def test_change_clb_node(self):
 
         # all the common failures
         self.assert_parses_common_clb_errors(expected.intent, eff)
+
+
+class NovaClientTests(SynchronousTestCase):
+    """
+    Tests for Nova client functions, such as :obj:`set_nova_metadata_item`.
+    """
+    def _setup_for_set_nova_metadata_item(self):
```

Comment:
I guess idiomatically I would expect two different test suites (one for `set_nova_metadata_item` and one for `get_server_details`) , and these _setup methods would be `setUp` methods that assign attributes on `self`. Was this a conscious decision to avoid that pattern? I do think that `setUp` methods can often get pretty gross (and do too much wrt what each tests needs), so I would understand a desire to avoid them.

Just curious, not asking you to change anything :)